examples/dynamic-bridge-app/linux/main.cpp

```diff
@@ -85,8 +85,10 @@ struct CommonAttributeAccessInterface : public chip::app::AttributeAccessInterfa
     CHIP_ERROR Read(const chip::app::ConcreteReadAttributePath & aPath, chip::app::AttributeValueEncoder & aEncoder) override;
     CHIP_ERROR Write(const chip::app::ConcreteDataAttributePath & aPath, chip::app::AttributeValueDecoder & aDecoder) override;
 
-    void OnListWriteBegin(const chip::app::ConcreteAttributePath & aPath) override;
-    void OnListWriteEnd(const chip::app::ConcreteAttributePath & aPath, bool aWriteWasSuccessful) override;
+    CHIP_ERROR Read(const chip::app::ConcreteReadAttributePath & aPath, TLV::TLVWriter & writer);
```

Comment:
Is this override?

Reply:
This is not an override. The function lives here because I did not want to add it to `AttributeAccessInterface` in `src` for this use case.

This is the reason the cast mentioned in https://github.com/project-chip/connectedhomeip/pull/23840#discussion_r1067398992 is needed.